index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aprendiendo Flexbox</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <img src="images/Imagen2.png" alt="logo" width="111px" height="74px"> 
+        <div class="boton">≡</div>
```

Comment:
- Hey great job on implementing the changes requested @fherrerao , however you did it but you're supposed to still have the images wrapped in the `nav` as you did before.
```
 <nav>
        <div class="espacio">-------</div>       
        <div class="boton">≡</div>

    </nav>
```

- Since this is just an omission, I'd mark it as OPTIONAL but kindly fix it before you merge :pray:

Reply:
Hi @zieeco ,
I did it, please review
Thank you